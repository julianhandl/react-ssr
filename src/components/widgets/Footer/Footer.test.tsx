import React from "react";
import { mount } from "enzyme";
import { Footer } from ".";
import { MemoryRouter } from "react-router";

describe("Footer", () => {
    const wrapper = mount(<MemoryRouter>
        <Footer />
    </MemoryRouter>)

    it("should render correctly", () => {
        expect(wrapper.find(Footer)).toMatchSnapshot();
    });
});