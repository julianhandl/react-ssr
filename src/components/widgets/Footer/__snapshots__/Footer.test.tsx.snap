// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<Footer>
  <footer
    className="footer"
  >
    <div
      className="footer__headline"
    >
      <div
        className="container"
      >
        <div
          className="footer__headline-items"
        >
          <div
            className="footer__headline-item"
          >
            <Link
              replace={false}
              to="/kontakt"
            >
              <a
                href="/kontakt"
                onClick={[Function]}
              >
                Musik
              </a>
            </Link>
          </div>
          <div
            className="footer__headline-item"
          >
            <Link
              replace={false}
              to="/kontakt"
            >
              <a
                href="/kontakt"
                onClick={[Function]}
              >
                Zelte
              </a>
            </Link>
          </div>
          <div
            className="footer__headline-item"
          >
            <Link
              replace={false}
              to="/kontakt"
            >
              <a
                href="/kontakt"
                onClick={[Function]}
              >
                Wissen
              </a>
            </Link>
          </div>
        </div>
      </div>
    </div>
    <div
      className="footer__main"
    >
      <div
        className="container"
      >
        <div
          className="footer__main-items"
        >
          <div
            className="footer__main-item"
          >
            <Link
              replace={false}
              to="/kontakt"
            >
              <a
                href="/kontakt"
                onClick={[Function]}
              >
                <h2>
                  Musik
                </h2>
              </a>
            </Link>
            <ul>
              <li
                key="footer-musik-klein"
              >
                <Link
                  replace={false}
                  to="/kontakt?paket=klein"
                >
                  <a
                    href="/kontakt?paket=klein"
                    onClick={[Function]}
                  >
                    Klein
                  </a>
                </Link>
              </li>
              <li
                key="footer-musik-mittel"
              >
                <Link
                  replace={false}
                  to="/kontakt?paket=mittel"
                >
                  <a
                    href="/kontakt?paket=mittel"
                    onClick={[Function]}
                  >
                    Mittel
                  </a>
                </Link>
              </li>
              <li
                key="footer-musik-gross"
              >
                <Link
                  replace={false}
                  to="/kontakt?paket=gross"
                >
                  <a
                    href="/kontakt?paket=gross"
                    onClick={[Function]}
                  >
                    Groß
                  </a>
                </Link>
              </li>
            </ul>
          </div>
          <div
            className="footer__main-item"
          >
            <Link
              replace={false}
              to="/kontakt"
            >
              <a
                href="/kontakt"
                onClick={[Function]}
              >
                <h2>
                  Zelte
                </h2>
              </a>
            </Link>
            <ul>
              <li
                key="footer-zelt-3x4.5"
              >
                <Link
                  replace={false}
                  to="/kontakt?paket=3x4.5"
                >
                  <a
                    href="/kontakt?paket=3x4.5"
                    onClick={[Function]}
                  >
                    3m x 4,5m
                  </a>
                </Link>
              </li>
              <li
                key="footer-zelt-12x5"
              >
                <Link
                  replace={false}
                  to="/kontakt?paket=12x5"
                >
                  <a
                    href="/kontakt?paket=12x5"
                    onClick={[Function]}
                  >
                    12m x 5m
                  </a>
                </Link>
              </li>
            </ul>
          </div>
          <div
            className="footer__main-item"
          >
            <Link
              replace={false}
              to="/wissen"
            >
              <a
                href="/wissen"
                onClick={[Function]}
              >
                <h2>
                  Wissen
                </h2>
              </a>
            </Link>
            <ul>
              <li>
                <Link
                  replace={false}
                  to="/kontakt"
                >
                  <a
                    href="/kontakt"
                    onClick={[Function]}
                  >
                    Beratung
                  </a>
                </Link>
              </li>
              <li>
                <Link
                  replace={false}
                  to="/kontakt"
                >
                  <a
                    href="/kontakt"
                    onClick={[Function]}
                  >
                    Begriffe
                  </a>
                </Link>
              </li>
              <li>
                <Link
                  replace={false}
                  to="/kontakt"
                >
                  <a
                    href="/kontakt"
                    onClick={[Function]}
                  >
                    Blog
                  </a>
                </Link>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      className="footer__bottom"
    >
      <div
        className="container"
      >
        <div
          className="footer__bottom-items"
        >
          <div
            className="footer__bottom-kontakt"
          >
            <div>
              <span>
                Julian Handl
              </span>
              <span>
                Dreihäuslweg 2
              </span>
              <span>
                3253 Erlauf
              </span>
            </div>
            <div>
              <a
                href="mailto:office@partypartner.at"
              >
                office@partypartner.at
              </a>
            </div>
            <div>
              <a
                href="tel:+4367763167169"
              >
                +43 677 631 671 69
              </a>
            </div>
          </div>
          <div
            className="footer__bottom-links"
          >
            <div>
              <span>
                © Copyright 
                2019
              </span>
              <span>
                Julian Handl
              </span>
            </div>
            <div>
              <Link
                replace={false}
                to="/datenschutz"
              >
                <a
                  href="/datenschutz"
                  onClick={[Function]}
                >
                  Datenschutz
                </a>
              </Link>
              <Link
                replace={false}
                to="/impressum"
              >
                <a
                  href="/impressum"
                  onClick={[Function]}
                >
                  Impressum
                </a>
              </Link>
              <Link
                replace={false}
                to="/agb"
              >
                <a
                  href="/agb"
                  onClick={[Function]}
                >
                  AGB
                </a>
              </Link>
            </div>
          </div>
        </div>
      </div>
    </div>
  </footer>
</Footer>
`;
